kratos/containers/csr_matrix.h

```diff
@@ -108,6 +108,26 @@ class CsrMatrix final
         SetValue(0.0);
     }
 
+    CsrMatrix(const MatrixMapType& matrix_map)
+    {
+        SparseGraph<TIndexType> Agraph;
+        for(const auto item : matrix_map){
```

Comment:
```suggestion
        for(const auto& item : matrix_map){
```
?, I don't know if this is a gain

Reply:
good catch!